cmd/launcher/extension.go

```diff
@@ -89,6 +93,7 @@ func createExtensionRuntime(ctx context.Context, rootDirectory string, db *bolt.
 		runtime.WithOsqueryExtensionPlugin(osquerylogger.NewPlugin("kolide_grpc", ext.LogString)),
 		runtime.WithOsqueryExtensionPlugin(distributed.NewPlugin("kolide_grpc", ext.GetQueries, ext.WriteResults)),
 		runtime.WithOsqueryExtensionPlugin(table.LauncherIdentifierTable(db)),
+		runtime.WithOsqueryExtensionPlugin(tm.Plugin()),
```

Comment:
This table should be registered via the normal `table.PlatformTables` mechanism in the runtime, not instantiated in main. See this for more info on where this happens: https://github.com/kolide/launcher/blob/ac1a399d3e0dbdb894b1e3ea41c0fc5aeb1e8d8d/pkg/osquery/runtime/runtime.go#L547-L549

Reply:
i talked through this with victor, and we came to a slightly different approach - the tables are launcher specific, and not related to platform functionality. I've abstracted them into `LauncherTables` which are loaded alongside launcher plugins